.github/scripts/review_pr.py

```
@@ -2,6 +2,9 @@
 import openai
 import requests
 import re
+import json
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)